jl/server.jl

```diff
@@ -0,0 +1,48 @@
+using JuliaParser,Lint
+import JSON
+
+include("utils.jl")
+include("symbols.jl")
+include("diagnostics.jl")
+include("completions.jl")
+include("definitions.jl")
+include("hovers.jl")
+
+const importcmdcache = Set{AbstractString}()
+
+const handlers = Dict(
+    "completions" => completionrequesthandler,
+    "definitions" => definitionrequesthandler,
+    "documents" => documentationrequesthandler,
+    "symbols" => symbolrequesthandler,
+    "diagnostics" => diagnosticshandler
+)
+
+function run()
+    while true
+        request = nothing
+        ok = false
+        try
+            request = JSON.parse(readline())
```

Comment:
This doesn't seem to matter: things printed to STDOUT not fitting the expected form of a ?Result don't get picked up by the TS side of things.